adapters/adapters-base/src/main/java/org/apache/cassandra/sidecar/adapters/base/CassandraMetricsOperations.java

```diff
@@ -61,14 +61,13 @@ public class CassandraMetricsOperations implements MetricsOperations
     private final ConnectedClientStatsDatabaseAccessor dbAccessor;
     protected final JmxClient jmxClient;
 
-
     /**
      * Creates a new instance with the provided {@link CQLSessionProvider}
      */
-    public CassandraMetricsOperations(JmxClient jmxClient, CQLSessionProvider session)
+    public CassandraMetricsOperations(JmxClient jmxClient, TableSchemaFetcher tableSchemaFetcher, ICassandraAdapter cassandraAdapter)
```

Comment:
`CassandraMetricsOperations` are targeting local node. We almost do not want to connect to other nodes from within the operations